### DISCUSSION TITLE
Video caching (RAM memory, Redis, other...)

### DISCUSSION THREAD

**Discussion by Dryadwoods** [jellyfin/jellyfin-meta#37] [Ideas]
Is this system at present day offering any kind of video caching (RAM memory, Redis, other...)?

Example: 
I start to see the video now, the video is in HDD#3, it is loaded into memory cache... moments later I pause the video, and on the following day (without restarting the server),  I try to resume the video. 
Or perhaps another user also wants to see the same video.
In both cases the video is loaded from memory and not from disk.

The objectives are plenty: not only for speed, but for trying to minimize the use of the HDD.

**Comment by nielsvanvelzen** [comment]
Please submit feature requests to https://features.jellyfin.org
